tools/toolchain/Dockerfile

```diff
@@ -12,6 +12,10 @@ RUN dnf -y update \
     && echo 'ALL ALL=(ALL:ALL) NOPASSWD: ALL' >> /etc/sudoers \
     && cp system-auth /etc/pam.d \
     && echo 'Defaults !requiretty' >> /etc/sudoers
+
+ADD ./tools/toolchain/clang_lto_pgo.sh .
+# Uncomment the following line to create a LTO+PGO'ed clang build
+# RUN chmod +x ./clang_lto_pgo.sh && ./clang_lto_pgo.sh && rm -rf /optimized_clang
```

Comment:
Either we do it, or we don't. Having every developer uncomment a line and build their own toolchain means many will miss out, and the rest will spend time compiling.

Reply:
+1

Developers don't build the dbuild container.
If we want it to be opt-in we can install it to a non-standard path (`/usr/bin/optimized-clang`) and then whoever wants to opt-in can `configure.py --compiler /usr/bin/optimized-clang`.

Reply:
So we want to have that optimized clang as part of the dbuild Dockerfile build process?